brave/src/main/java/brave/http/HttpClientHandler.java

```diff
@@ -12,35 +13,25 @@ public HttpClientHandler(HttpAdapter<Req, Resp> adapter, HttpClientParser parser
     this.parser = parser;
   }
 
-  public Req handleSend(Req request, Span span) {
-    if (span.isNoop()) return request;
+  public void handleSend(Req request, Span span) {
```

Comment:
these return values weren't used, or when used encouraged people to use a span in scope.